### ISSUE TITLE
[UX][Improvement]  Do not change to active subscribers from unreachable contacts when unreachable are listed

### ISSUE THREAD

**Issue by digitalfredy** [pentacent/keila#193]
Hello,

# Context
I migrate a base of contacts to Keila and when I send my first newsletter I got around 70 unreachable contacts then I decided to cleanup them and I found those 70 contacts at `/contacts/unreachable` distributed in two pages.

# The problem
I select all the contacts in the first page and then I delete them and I selected again all contacts and I was going to delete all of them thinking it was the page 2 for `/contacts/unreachable` but fortunately I notice an email I know  and I realized I was going to delete a full page of active contacts !!!!

# Proposed experience
I think it makes sense to move from `/contacts/unreachable`  to `/contacts`   **only if there is no more unreachable contacts in the list including other pages** otherwise keep  `/contacts/unreachable` because this way as user I could continue deleting unreachable users.

**Comment by wmnnd** [comment]
Thanks for reporting this, I’ll see how this can be fixed!

**Comment by digitalfredy** [comment]
Hi @wmnnd 

Today I had the same issue and while searching for unreachable I found this bug I did myself.

Will be great to dedicate some time to this matter because almost sure many people already delete contacts that they do not wanted to delete.

Again ,the issue is if I go to unreachable, I select all in the first page while having more that one page and I chose delete, after I delete I will be redirected to /contacts that contains the contacts I do not want to delete. The expected UX is to continue in the same palace where I was (unreachable).
